File: _app/src/main/java/demo/com/demolibraries/main/MenuID.java
package demo.com.demolibraries.main;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({
        MenuID.DATA_HANDLER,
        MenuID.SENSORS,
        MenuID.MEDIA_PLAYER,
        MenuID.FACEBOOK,
        MenuID.CUSTOM_VIEW,
        MenuID.TEST_CALL,
        MenuID.SPEECH_RECOGNITION
})
@Retention(RetentionPolicy.SOURCE)
@interface MenuID {
    int DATA_HANDLER = 0;
    int SENSORS = 1;
    int MEDIA_PLAYER = 2;
    int FACEBOOK = 3;
    int CUSTOM_VIEW = 4;
    int TEST_CALL = 5;
    int SPEECH_RECOGNITION = 6;
}

File: _app/src/main/java/demo/com/demolibraries/main/MenuID.kt
package demo.com.demolibraries.main

import android.support.annotation.IntDef

const val DATA_HANDLER = 0
const val SENSORS = 1
const val MEDIA_PLAYER = 2
const val FACEBOOK = 3
const val CUSTOM_VIEW = 4
const val TEST_CALL = 5
const val SPEECH_RECOGNITION = 6
const val BLUETOOTH = 7
const val CHAT = 7

@IntDef(DATA_HANDLER, SENSORS, MEDIA_PLAYER, FACEBOOK, CUSTOM_VIEW, TEST_CALL, SPEECH_RECOGNITION, BLUETOOTH, CHAT)
@Retention(AnnotationRetention.SOURCE)
annotation class MenuID